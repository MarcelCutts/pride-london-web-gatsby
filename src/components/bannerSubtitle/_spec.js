import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { shallow } from 'enzyme'

import StyledBannerSubtitle from './'

describe('BannerSubtitle', () => {
  it('should render', () => {
    const wrapper = shallow(<StyledBannerSubtitle />)
    expect(wrapper).toMatchSnapshot()
  })

  it('should render a Wrapper', () => {
    const wrapper = shallow(<StyledBannerSubtitle />)
    expect(wrapper.find('SubtitleWrapper')).toHaveLength(1)
  })

  it('should render a Text', () => {
    const wrapper = shallow(<StyledBannerSubtitle />)
    expect(wrapper.find('SubtitleText')).toHaveLength(1)
  })

  it('should render child text ', () => {
    const text = 'And here is a test subtitle!'
    const wrapper = shallow(<StyledBannerSubtitle>{text}</StyledBannerSubtitle>)
    expect(wrapper).toMatchSnapshot()
  })
})
