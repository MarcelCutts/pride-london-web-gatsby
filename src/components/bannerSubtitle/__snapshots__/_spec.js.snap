// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerSubtitle should render 1`] = `
.c0 {
  color: #ffffff;
  font-size: 16px;
}

.c0 span {
  background-color: #2d2f7f;
  display: inline-block;
  padding: 0 10px;
}

<h3
  className="c0"
>
  <span />
</h3>
`;
